engine/verification/fetcher/engine.go

```diff
@@ -350,6 +350,15 @@ func (e *Engine) requestChunkDataPack(c *ChunkStatus, allExecutors flow.Identity
 
 	targetIDs := chooseChunkDataPackTarget(allExecutors, c.Agrees, c.Disagrees)
 
+	// TEMP: skip sending chunk data pack requests to avoid overloading ENs
+	// This is OK because RAs are not yet used in the sealing process.
+	e.log.Warn().
+		Hex("block_id", c.Chunk.BlockID[:]).
+		Uint64("chunk_index", c.Chunk.Index).
+		Msg("skipping sending chunk data pack request")
+
+	return nil
```

Comment:
Does this compile, I think the code after is reported as unreachable no?

Reply:
It compiles